### PR TITLE
included backend support for Video in FCM

### PR DESCRIPTION
**Pull request by kave**
Short description explaining the high-level reason for the pull request

## Additions

- included backend support for Video in FCM

## Testing

- Create/Edit a sublanding page and add video related fields

## Review

- @KimberlyMunoz 
- @kurtw 
- @richaagarwal 

## Screenshots
screen shot 2016-03-11 at 1 21 27 pm

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by KimberlyMunoz** [comment]
Made a few changes on the frontend to make this work in 11ddabb.

**Comment by kave** [comment]
:+1:

File: cfgov/unprocessed/css/cf-enhancements.less [hunk 1]
```diff
@@ -1702,6 +1702,12 @@ select[multiple] {
     .respond-to-max(@fcm-bp-max, {
         margin-bottom: @grid_gutter-width;
     });
+
+    .video-player_image {
+      max-width: none;
+      height: 320px;
```
Comment (sebworks) [inline cfgov/unprocessed/css/cf-enhancements.less]: Why do we need to set the height?

File: cfgov/unprocessed/css/cf-enhancements.less [hunk 2]
```diff
@@ -1702,6 +1702,12 @@ select[multiple] {
     .respond-to-max(@fcm-bp-max, {
         margin-bottom: @grid_gutter-width;
     });
+
+    .video-player_image {
+      max-width: none;
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/cf-enhancements.less]: Update to 4 space indentation (just for Less files)

**Comment by sebworks** [comment]
:+1:

File: cfgov/unprocessed/css/cf-enhancements.less (hunk 1 above)
Comment (KimberlyMunoz) [inline cfgov/unprocessed/css/cf-enhancements.less]: It resizes the screenshot for the video to be the right size.

File: cfgov/unprocessed/css/cf-enhancements.less (hunk 2 above)
Comment (KimberlyMunoz) [inline cfgov/unprocessed/css/cf-enhancements.less]: Fixed in 6892eb0

File: cfgov/unprocessed/css/cf-enhancements.less (hunk 1 above)
Comment (sebworks) [inline cfgov/unprocessed/css/cf-enhancements.less]: :+1:

**Pull request merged**